src/keys/rsa.js

```diff
@@ -1,43 +1,23 @@
 'use strict'
 
 const crypto = require('crypto')
+const promisify = require('util').promisify
 const errcode = require('err-code')
 const randomBytes = require('../random-bytes')
-// @ts-check
-/**
- * @type {PrivateKey}
- */
-let keypair
-try {
-  if (process.env.LP2P_FORCE_CRYPTO_LIB === 'keypair') {
-    throw new Error('Force keypair usage')
-  }
-
-  const ursa = require('ursa-optional') // throws if not compiled
-  keypair = ({ bits }) => {
-    const key = ursa.generatePrivateKey(bits)
-    return {
-      private: key.toPrivatePem(),
-      public: key.toPublicPem()
-    }
-  }
-} catch (e) {
-  if (process.env.LP2P_FORCE_CRYPTO_LIB === 'ursa') {
-    throw e
-  }
-
-  keypair = require('keypair')
-}
-const pemToJwk = require('pem-jwk').pem2jwk
-const jwkToPem = require('pem-jwk').jwk2pem
+const keypair = promisify(crypto.generateKeyPair)
 
 exports.utils = require('./rsa-utils')
 
 exports.generateKey = async function (bits) { // eslint-disable-line require-await
-  const key = keypair({ bits })
+  const key = await keypair('rsa', {
```

Comment:
Could use the sync version here, but used the async version seeing as how the function is async anyway.